pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -516,6 +516,11 @@ buildStdenv.mkDerivation ({
     LDFLAGS = "-Wl,-rpath,${placeholder "out"}/lib/${binaryName}";
   };
 
+  # workaround for clang statically linking against libstdc++
```

Comment:
Might be worth mentioning why this is a problem and needs a workaround, and/or link to the issue

Reply:
We will immediately revert this commit, since the proper fix is in https://github.com/NixOS/nixpkgs/pull/267737.